--- lalinference-4.1.1/Makefile.am.orig	2022-05-11 00:56:41.829041103 +0200
+++ lalinference-4.1.1/Makefile.am	2023-02-11 07:56:42.424597083 +0100
@@ -25,11 +25,6 @@ SUBDIRS = \
 user_environment = \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
