--- lalinference-4.1.6/Makefile.am.orig	2024-06-09 07:55:13.554955545 +0200
+++ lalinference-4.1.6/Makefile.am	2024-06-09 07:55:45.034785004 +0200
@@ -26,11 +26,6 @@ user_environment = \
 	trailingcolon MANPATH; \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
